docs: Fix conflicting slug in docs

grouville opened the pull request:
Docs have been following an incremental `id/name` slug policy from the start. The id shall be represented on all the incoming docs (with a few exceptions) and shall not conflict with each other. Fixes the current conflicts

Signed-off-by: guillaume

TomChv reviewed (commented):
I wrote a little command to find all duplicate tags in markdown.

```shell
for f in $(find . -name "1*-*.md"); do cut -d'-' -f1 <<< $(basename $f) >> dup.txt; done; sort dup.txt | uniq -cd
```

It produces the following result on main:

```
   2 1227
   2 1229
   2 1240
   3 1241
```

And the following result on your branch:

```
 2 1229
```

So I think you missed a duplicate on files:
```
1229-go-ci.md
1229-empty-buildkit-cache.md
```

Feel free to also fix conflict on this one then merge ;)

grouville commented:
> I wrote a little command to find all duplicate tags in markdown.
> 
> ```shell
> for f in $(find . -name "1*-*.md"); do cut -d'-' -f1 <<< $(basename $f) >> dup.txt; done; sort dup.txt | uniq -cd
> ```
> 
> It produces the following result on main:
> 
> ```
>    2 1227
>    2 1229
>    2 1240
>    3 1241
> ```
> 
> And the following result on your branch:
> 
> ```
>  2 1229
> ```
> 
> So I think you missed a duplicate on files:
> 
> ```
> 1229-go-ci.md
> 1229-empty-buildkit-cache.md
> ```
> 
> Feel free to also fix conflict on this one then merge ;)

Indeed, thanks 😇. Rebased and fixed

gerhard commented:
When we make changes like this one, we will need to kick off the [Docsearch crawler](https://docsearch.algolia.com/) so that it rebuilds the search index correctly. If we do nothing, it rebuilds every 24h automatically, which is OK. I've just hit this issue now and manually triggered the re-crawl via https://crawler.algolia.com/admin/crawlers/0941f59c-ccd3-4635-8424-72e3f5017147/overview. Mentioning it here in case someone else hits it. FWIW #2348
